src/boot/efi/util.c

```diff
@@ -105,26 +105,19 @@ EFI_STATUS chunked_read(EFI_FILE *file, size_t *size, void *buf) {
         return EFI_SUCCESS;
 }
 
-EFI_STATUS file_read(
```

Comment:
Hmm, why not just permit `name` to be NULL?

Reply:
You mean, `file_read()` should also take `handle`?? I prefer the current form.